package.json

```diff
@@ -95,7 +96,6 @@
     "worker-loader": "^0.7.0"
   },
   "dependencies": {
-    "highlight-words-core": "^1.1.2",
```

Comment:
This seems accidental. Revert?

Reply:
Done. Just remember that the changes in the new react-highlight-words require the new core package as dependency. So you'll need to update the version here before publishing.